### PR TITLE
chore: strip service name prefix from enums

### PR DESCRIPTION
**Pull request by hamidzr**
## Description

- update swagger post-processing to remove service name prefix from
the remaining model definitions, mainly enums
- set `swag` to format on `make -C master fmt`

## Commentary (optional)

includes a batch sed for
```
Determinedcheckpointv1State Checkpointv1State
Determinedcontainerv1State Containerv1State
Determineddevicev1Type Devicev1Type
Determinedexperimentv1State Experimentv1State
Determinedjobv1State Jobv1State
Determinedjobv1Type Jobv1Type
Determinedtaskv1State Taskv1State
Determinedtrialv1State Trialv1State
determinedcheckpointv1State checkpointv1State
determinedcontainerv1State containerv1State
determineddevicev1Type devicev1Type
determinedexperimentv1State experimentv1State
determinedjobv1State jobv1State
determinedjobv1Type jobv1Type
determinedtaskv1State taskv1State
determinedtrialv1State trialv1State
```

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

**Review by rb-determined-ai — APPROVED** [review]
I love it!

**Review by keita-determined — APPROVED** [review]
did ad hoc test for webUI on local and LGTM!

qq: 
i see some states with `determined` in swagger. Is it intentional? 

image

**Comment by hamidzr** [comment]
thanks for the reviews

> did ad hoc test for webUI on local and LGTM!
> 
> qq: i see some states with `determined` in swagger. Is it intentional?
> 
> image

good catch there was another field that's apparently just used for documentation `title` that I wasn't updating. I updated it and it should look like this now 
image

File: proto/scripts/swagger.py [hunk 1]
```diff
@@ -50,9 +50,12 @@ def clean(path: str, patch: str) -> None:
         elif key == "protobufNullValue":
             value["title"] = "NullValue"
 
-        # Clean up titles.
+        # Clean up titles. Title is used in documentation.
         if "title" not in value:
             value["title"] = "".join(capitalize(k) for k in key.split(sep="v1"))
+        elif value["title"].startswith(SERVICE_NAME):
```
Comment (hamidzr) [inline proto/scripts/swagger.py:56]: the way titles are generated seems to be different for enums vs messages defined in proto.  in our current setup, this check precisely targets the enums we want but we could maybe add a regex to skip titles like `Determined abc` or assert that this is not the case

**Pull request merged**